bin/citgm-all.cjs

```diff
@@ -1,16 +1,9 @@
 #!/usr/bin/env node
 
-const { spawnSync } = require('child_process');
-const { join } = require('path');
-
 const supportsESM = process.version.startsWith('v14.');
```

Comment:
this will be more accurate and robust
```suggestion
const supportsESM = require('has-package-exports/conditional');
```

Reply:
that's temporary. I'm working with @MylesBorins on https://github.com/targos/supports-esm

Reply:
indeed, but that implementation uses user-agent sniffing, doing version comparison. Since conditional exports are in lockstep with "full ESM" support, and `has-package-exports` actually uses feature detection, it will be robust in a way that `supports-esm` is currently not.

You can actually replace the entire implementation of `supports-esm` with `require('has-package-exports/conditional')`, and it won't rely on version sniffing or node flag inspection.